src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -333,26 +408,39 @@ export { ActiveFocusInvestigation };
 
 /** interface to describe props from mapStateToProps */
 interface DispatchedStateProps {
+  plan: Plan | null;
   caseTriggeredPlans: Plan[] | null;
   routinePlans: Plan[] | null;
 }
 
 /** map state to props */
-const mapStateToProps = (state: Partial<Store>): DispatchedStateProps => {
+const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateProps => {
+  const planId =
+    ownProps.match.params && ownProps.match.params.plan_id ? ownProps.match.params.plan_id : null;
+  const plan = planId ? getPlanById(state, planId) : null;
+  const jurisdictionParentId =
+    ownProps.match.params && ownProps.match.params.jurisdiction_parent_id
+      ? ownProps.match.params.jurisdiction_parent_id
+      : null;
   const caseTriggeredPlans = getPlansArray(
     state,
     InterventionType.FI,
     [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
-    CASE_TRIGGERED
+    CASE_TRIGGERED,
+    [],
+    jurisdictionParentId
   );
   const routinePlans = getPlansArray(
     state,
     InterventionType.FI,
     [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
-    ROUTINE
+    ROUTINE,
+    [],
+    jurisdictionParentId
```

Comment:
Could you please fix the linting error here?

Reply:
These linting errors are already fixed